client/my-sites/checkout/src/components/prepurchase-notices/cart-item-is-legacy-bundle-upgrade.tsx

```diff
@@ -0,0 +1,37 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import PrePurchaseNotice from './prepurchase-notice';
+import type { ResponseCartProduct } from '@automattic/shopping-cart';
+import type { SitePlan } from 'calypso/state/sites/selectors/get-site-plan';
+
+type Props = {
+	sitePlan: SitePlan;
+	cartProduct: ResponseCartProduct;
+};
+
+const SitePlanIsLegacyBundleUpgrade: FunctionComponent< Props > = ( { sitePlan, cartProduct } ) => {
+	const translate = useTranslate();
+	const supportUrl = localizeUrl( 'https://jetpack.com/contact-support/' );
+	const message = translate(
+		'The %(product)s plan will replace your %(existingPlan)s plan. %(product)s is packed with additional features, but does have a lower backup storage capacity. Please contact support for a free storage upgrade for one year.',
```

Comment:
Some commentary here might be useful here to explain why do we assume newer bundles have "lower backup storage capacity"